Feature/quotas

jeanCodeUnMax opened the pull request.

jeanCodeUnMax committed:
:sparkles: Composant Quotas

mise en place avec props  activation sur DetailCourse  <quotas value{xxx}/>

jeanCodeUnMax committed:
:sparkles: Mise a jour commit composant quotas

j'ai changer de container wsl avant de commit et je pense qu un dossier a ete oublie

ponche commented on components/DetailCourse.tsx:
```diff
@@ -4,6 +4,8 @@ import ChronoTopDepart from "./ChornoTopDepart";
 import DetailTrajet from "./DetailTrajet";
 import MenuBurger from "./MenuBurger";
 import CardNumeroLine from "./CardNumeroLine";
+import Quotas from "./Quotas";
+import styles from "./modals/modalStyle";
```
Pourquoi tu importe Quotas et modalStyle dans le ChronoTopDepart ?

ponche commented on components/DetailCourse.tsx:
```diff
@@ -24,9 +26,12 @@ export default function DetailCourse() {
       </View>
-      <View style={style.line}>
-        <CardNumeroLine lineNumber="K12" />
-        <Text variant="headlineSmall">842526</Text>
```
Pourquoi tu supprime mon style ? 😡

ponche commented on components/Quotas.tsx:
```diff
@@ -0,0 +1,23 @@
+    <View>
+      <Text>
+        {"  "}Objectif{"  "}
+        <Icon name="receipt-outline" size={40} color="#000" />
+        {""}
+        {value}
+        {"  "}
```
Icon et value ne sont pas proportionné. 
Icon est trop grand par rapport au texte.

ponche reviewed (changes requested):
Icon et le text n'est pas proportionné. Icon est trop grand par rapport au Texte. 

Maintenant, l’enquêteur  devra prendre des loupe pour lire le texte. 😠 

l'idée de mettre un Icon, c'est de ne pas mettre le texte:  *objectif*

jeanCodeUnMax committed:
:lipstick: Mise a jour composant Quotas

mise en place Quotas  avec modification DetailCourse structure des div pour le css !

jeanCodeUnMax commented:
mise a jour terminé
